File: src/types/resources.ts
export type ResourceCategory = 'Food & Rations' | 'Water' | 'Medicine' | 'Fuel' | 'Generators' | 'Medical Kits' | 'Equipment';

export interface ResourceItem {
  id: string;
  code: string;
  name: string;
  category: ResourceCategory;
  currentStock: number;
  minThreshold: number;
  unit: string;
  storageHub: string;
  status: 'OPTIMAL' | 'LOW STOCK' | 'CRITICAL EMPTY';
  lastReplenished: string;
}

File: backend/app/api/v1/resources.py
from fastapi import APIRouter

router = APIRouter()

RESOURCES_DB = [
    {
        "id": "res_1",
        "name": "Potable Drinking Water Drums (200L)",
        "category": "water",
        "currentStock": 420,
        "minThreshold": 1000,
        "maxCapacity": 3000,
        "unit": "Drums",
        "storageHub": "Central Logistics Depot A",
        "status": "LOW STOCK"
    }
]

@router.get("")
def list_resources():
    return {"resources": RESOURCES_DB}
